Testscripts/Linux/fpga-card-validation.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+########################################################################
+#
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+#
+########################################################################
+########################################################################
+#
+# Description:
+#   This script assumes Xilinx XRT tools are available on the VM
+#   and tries to run the validation test to make sure FPGA cards
+#   are functional. 
+#
+#
+########################################################################
+
+#######################################################################
+function prepare() {
+    xrt_setup="/opt/xilinx/xrt/setup.sh"
```

Comment:
is 'setup.sh' a built-in script in FPGA? or, is it common in FPGA? This should be accessible if setup.sh is used.

Reply:
`setup.sh` comes from [Xilinx XRT package](https://github.com/Xilinx/XRT). I assume the `.vhd` image has all the required packages.

Reply:
I am not sure everyone runs this test with "vhd" has the package installed. 
Now I have a few questions as the preliminary condition.
1. Does this test need VHD? Does it work with any Azure image? Like you see README, LISAv2 support Azure, Hyper-V and WSL. You define 'Azure' in Platform tag, so it should work with Azure image.  My question is if you are using VHD file not Azure Gallery image.
2. If you see other test scripts, the test script checks out if the tool or dependency is available or not. Otherwise, the test script installs required packages and dependency. This should be done in your test. Do not assume all have the same VHD file.

Reply:
The Xilinx XRT tool is supported under CentOS 7.6 or Ubuntu 18.04 with kernel < 4.18.x. So, the user cannot use any Azure image to deploy the VM. To make the testing process faster (avoiding the installation of the tool every time I want to do the test), I create the `VHD` using Shared Image Gallery and rebuild the VHD when the XRT tool is updated.

Reply:
Please change to hyperv in <Platform>Azure</Platform>
Also you will need to set distro version and kernel version filter like https://github.com/LIS/LISAv2/blob/master/Testscripts/Linux/Linux-Kernel-Self-Tests.sh#L78

Reply:
Here is @LiliDeng comment - [Lili] We’d better let him automation the steps which he did in the prepare image steps. But it is not that important, since gallery image will never run this case since it needs an older version kernel